packages/uikit/src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -134,6 +130,11 @@ const DropdownMenu: React.FC<React.PropsWithChildren<DropdownMenuProps>> = ({
                         {...itemProps}
                       >
                         {MenuItemContent}
+                        {label === "Buy Crypto" ? (
+                          <Button height="25px" variant="secondary" px="8px">
+                            New
+                          </Button>
+                        ) : null}
```

Comment:
Why should we add the `New` tag in here?
U can set in at menu [config](https://github.com/pancakeswap/pancake-frontend/blob/31e276842ebc315a15ecbc5d2df3e73a6220a53f/apps/web/src/components/Menu/config/config.ts#L173).